week-2-practise/level-3/5-indexOf.js

```diff
@@ -6,7 +6,8 @@
 // - return a new array with the replacement value inserted
 
 function replace(arr, valueToReplace, newValue) {
-  return; // complete this statement
+  newArr = Object.assign([], arr, {[arr.indexOf(valueToReplace)]:newValue});
```

Comment:
The same advice about missing variable declarations and not using `Object.assign` with array applies here too.